README.md

```diff
@@ -1,83 +1,114 @@
-# Cheesecake Checklist 
+# Cheesecake Checklist
 
-## CKL
 ### Best Pratices
-- [ ] README
+- [ ] [README]()
 - [ ] [Dependabot]()
+
 ### Keeper
-- [ ] AWS credentials
-- [ ] Google credentials
-- [ ] Terraform env vars
+- [ ] [AWS credentials]()
+- [ ] [Google credentials]()
+- [ ] [Terraform env vars]()
+
 ### GDPR
-- [ ] Encrypt, pseudonymize, or anonymize personal data wherever possible
-- [ ] Sign a data processing agreement between your organization and any third parties that process personal data on your behalf
-- [ ] It's easy for your customers to request and receive all the information you have about them
-- [ ] It's easy for your customers to request to have their personal data deleted
+- [ ] [Encrypt, pseudonymize, or anonymize personal data wherever possible]()
+- [ ] [Sign a data processing agreement between your organization and any third parties that process personal data on your behalf]()
+- [ ] [It's easy for your customers to request and receive all the information you have about them]()
+- [ ] [It's easy for your customers to request to have their personal data deleted]()
+
 ### Quality
-- [ ] CodeClimate for the project and enable Pull Requests reviews
-- [ ] Sentry is configured on Google client's account
-- [ ] pre-commit gjira installed
+- [ ] [CodeClimate for the project and enable Pull Requests reviews]()
+- [ ] [Sentry is configured on Google client's account]()
+- [ ] [pre-commit gjira installed]()
 
 ## BACKEND
+
 ### Documentation
-- [ ] Entity
-- [ ] Infra
-- [ ] API docs/swagger
+
+- [ ] [Infrastructure diagram]()
+- [ ] [Entity-Relationship diagram]()
+- [ ] [Architecture definitions document]()
+- [ ] [Sentry]()
+- [ ] [Infrastructure setup]()
```

Comment:
is it about terraform?

Reply:
I thought about diagrams but could be the terraform config too.